Shoko.Server/Services/VideoLocal_PlaceService.cs

```diff
@@ -410,9 +410,10 @@ public MoveFileResult MoveFile(SVR_VideoLocal_Place videoLocalPlace, bool delete
 
             if (dropFolder.IsDropSource == 1 && deleteEmpty)
             {
-                var directories = dropFolder.BaseDirectory.EnumerateDirectories("*", new EnumerationOptions() { RecurseSubdirectories = true, ReturnSpecialDirectories = true, IgnoreInaccessible = true })
+                var directories = dropFolder.BaseDirectory.EnumerateDirectories("*", new EnumerationOptions() { RecurseSubdirectories = true, ReturnSpecialDirectories = false, IgnoreInaccessible = true })
                     .Select(dirInfo => dirInfo.FullName);
-                RecursiveDeleteEmptyDirectories(directories, dropFolder.ImportFolderLocation);
+                if (directories.Any())
```

Comment:
Revert the added if statement. we don't want to run the enumerator twice if we can avoid it, and the internals of the method can handle an empty enunerator just fine.